tests/queries/tests.py

```diff
@@ -3678,3 +3678,11 @@ def test_ticket_24278(self):
         School.objects.create()
         qs = School.objects.filter(Q(pk__in=()) | Q())
         self.assertQuerysetEqual(qs, [])
+
+
+class TestInvalidValuesRelation(TestCase):
+    def test_invalid_values(self):
+        with self.assertRaises(ValueError):
+            Annotation.objects.filter(tag='abc')
```

Comment:
IIRC raising an error in this case was only tested in one admin test, and it was hard to figure out what exactly was broken. I added this test so that we have a more focused test for this.